bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -284,6 +285,13 @@ private constructor(
     @get:JvmName("timelines")
     val timelines: TimelineMethods by lazy { TimelineMethods(this) }
 
+    /**
```

Comment:
Lines 288-294 (pushNotifications) should be moved right after the polls endpoint in this file to make it alphabetically ordered. Sorry if I was not explaining it correctly. :-)